install-php-extensions

```diff
@@ -55,7 +55,7 @@ getDistroVersion() {
 # Output:
 #   The PHP numeric Major-Minor version
 getPHPMajorMinor() {
-	php -r '$v = explode(".", PHP_VERSION); echo $v[0] * 100 + $v[1];'
+	php-config --version | awk -F. '{print $1*100+$2}'
```

Comment:
Any reason for this change?

Reply:
awk works faster for the first run (as php reads up all libraries/dependencies):

$ docker run -ti lakostis123/php7-fpm-alpine /bin/sh
/var/www/html # time php -r '$v = explode(".", PHP_VERSION); echo $v[0] * 100 + $v[1];'
703real 0m 0.07s
user    0m 0.01s
sys     0m 0.01s

/var/www/html # time php-config --version | awk -F. '{print $1*100+$2}'
703
real    0m 0.00s
user    0m 0.00s
sys     0m 0.00s

But of course it's up to you to decide.